dwave_networkx/examples/min_maximal_matching.py

```diff
@@ -13,7 +13,6 @@
 #    limitations under the License.
 #
 # ================================================================================================
-from __future__ import print_function
```

Comment:
Unfortunately we haven't disabled the CI for python 2.7, so this is failing.

Reply:
sounds good